### DISCUSSION TITLE
Want to add the Plausible in my frontend Code which is written in angular.

### DISCUSSION THREAD

**Discussion by Riteshag05** [plausible/analytics#4634] [Self-Hosted Support]
Can somebody send me code or help me how can i plausible in my angular code.

I want to add the screens of my frontend into Plausible, my supervisor asked me to do it from frontend and not adding the script in index.html file. Is there any piece of code or anything that will help me to add the screens in Plausible but it should be added in angular code

**Comment by ruslandoga** [comment]
👋 @Riteshag05 

I saw this recently: https://github.com/plausible/docs/issues/432

Note that it might be outdated and that approach might no longer work.
